crates/re_ui/src/command.rs

```diff
@@ -98,33 +98,31 @@ impl UICommand {
             ),
 
             UICommand::ToggleMemoryPanel => (
-                "Toggle memory panel",
-                "Investigate what is using up RAM in Rerun Viewer",
+                "Toggle Memory Panel",
+                "View and track current RAM usage inside Rerun Viewer",
             ),
-            UICommand::ToggleBlueprintPanel => ("Toggle blueprint panel", "Toggle the left panel"),
-            UICommand::ToggleSelectionPanel => ("Toggle selection panel", "Toggle the right panel"),
-            UICommand::ToggleTimePanel => ("Toggle time panel", "Toggle the bottom time panel"),
+            UICommand::ToggleBlueprintPanel => ("Toggle Blueprint Panel", "Toggle the left panel"),
+            UICommand::ToggleSelectionPanel => ("Toggle Selection Panel", "Toggle the right panel"),
+            UICommand::ToggleTimePanel => ("Toggle Time Panel", "Toggle the bottom panel"),
 
             #[cfg(not(target_arch = "wasm32"))]
             UICommand::ToggleFullscreen => (
                 "Toggle fullscreen",
                 "Toggle between windowed and fullscreen viewer",
             ),
             #[cfg(not(target_arch = "wasm32"))]
-            UICommand::ZoomIn => ("Zoom In", "Increases the ui scaling factor"),
+            UICommand::ZoomIn => ("Zoom In", "Increases the UI scaling factor"),
             #[cfg(not(target_arch = "wasm32"))]
-            UICommand::ZoomOut => ("Zoom Out", "Decreases the ui scaling factor"),
+            UICommand::ZoomOut => ("Zoom Out", "Decreases the UI scaling factor"),
             #[cfg(not(target_arch = "wasm32"))]
             UICommand::ZoomReset => (
                 "Reset Zoom",
-                "Resets ui scaling factor to the OS provided default",
+                "Resets UI scaling factor to the OS provided default",
```

Comment:
Suggestion: `Resets the zoom level to the operating system's default value`

(Don't introduce the new term `scaling factor`. Simplify the other language.)

Reply:
We need to fix the other two as well then:

<img width="602" alt="image" src="https://github.com/rerun-io/rerun/assets/49431240/5cdb771a-04bc-43cd-963a-da7012192231">

`Increase the zoom level` feels somewhat ambiguous. I've gone with `Increase the UI zoom level`, and changed the other one to `Resets the UI zoom level to the operating system's default value` for consistency.

Reply:
Good call, there could be many types of zoom levels. I say ship it! 😊